packages/rspack/src/builtin-plugin/SwcJsMinimizerPlugin.ts

```diff
@@ -39,24 +39,26 @@ function getMinifyCondition(
 	throw new Error("unreachable: condition should be one of string, RegExp");
 }
 
-function getMinifyConditions(
+function getRawSwcJsMinimizerRules(
 	condition?: MinifyConditions
-): RawMinificationConditions | undefined {
+): RawSwcJsMinimizerRules | undefined {
 	if (!condition) return undefined;
 
 	if (Array.isArray(condition)) {
 		return {
 			type: "array",
-			arrayMatcher: condition.map(i => getMinifyCondition(i))
+			arrayMatcher: condition.map(i => getRawSwcJsMinimizerRule(i))
 		};
 	}
 
-	return getMinifyCondition(condition);
+	return getRawSwcJsMinimizerRule(condition);
 }
 
-export const SwcJsMinimizerPlugin = create(
-	BuiltinPluginKind.SwcJsMinimizer,
-	(options?: SwcJsMinimizerPluginOptions): RawMinification => {
+export const SwcJsMinimizerRspackPlugin = create(
```

Comment:
breaking change note.

Is this the final version name? modern has used SwcJsMinimizerPlugin in https://github.com/web-infra-dev/modern.js/pull/4602, lucky we haven't released it yet.

Reply:
This is the final version name, there should be no one but modernjs uses SwcJsMinimizerPlugin, cause we didn't ship any docs or release note highlight about it in v0.3.2 intentionally, I consider it is stable to adopt after tomorrow's release (v0.3.3)

Reply:
okk

Reply:
> This is the final version name, there should be no one but modernjs uses SwcJsMinimizerPlugin, cause we didn't ship any docs or release note highlight about it in v0.3.2 intentionally, I consider it is stable to adopt after tomorrow's release (v0.3.3)

Would you add a final naming convention guide for future developers?

Reply:
Ofc